### PR TITLE
Updated phpversions, number of contributors and all datas about maintainers

### PR DESCRIPTION
**Pull request by fballiano**
Datas about maintainers is taken from https://github.com/OpenMage/magento-lts/blob/1.9.4.x/README.md and https://github.com/OpenMage/magento-lts/graphs/contributors

I do not have possibility to test the changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -352,38 +351,155 @@ <h3 class="contributor__subtitle">
...
-                                            Daniel Fahlke
+                                        <img class="maintainers__image" src="/assets/images/maintainers/lee.png" />
+                                        <div class="maintainers__name">Lee Saferite</div>
+                                    </div>
+    
+                                    <div class="maintainers__more-info">
+                                        <div class="maintainers__stats">
+                                            <div class="maintainers__stats-item">
+                                                <img class="maintainers__stats-icon" src="/assets/svg/commit.svg" />
+                                                <span class="maintainers__stats-text">75</span>
```
Comment (Flyingmana) [inline index.html:393]: why did it go down from 117?

File: index.html [hunk 2]
```diff
@@ -526,4 +642,4 @@ <h2 class="blog__title col">
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
```
Comment (Flyingmana) [inline index.html:645]: can you add an empty line at the end here?

File: index.html [hunk 3]
```diff
@@ -298,44 +292,49 @@ <h3 class="contributor__subtitle">
...
-                                        </div>
+                                        <div class="maintainers__name">David Robinson</div>
                                     </div>
     
                                     <div class="maintainers__more-info">
                                         <div class="maintainers__stats">
                                             <div class="maintainers__stats-item">
                                                 <img class="maintainers__stats-icon" src="/assets/svg/commit.svg" />
-                                                <span class="maintainers__stats-text">53</span>
+                                                <span class="maintainers__stats-text">31</span>
```
Comment (Flyingmana) [inline index.html:337]: why did it go down?

File: index.html [hunk 4]
```diff
@@ -395,7 +511,7 @@ <h3 class="contributor__subtitle">
                             <div class="row">
                                 <div class="col-xl-10 offset-xl-1">
                                     <div class="contributor__list-title">
-                                        Over 100 Contributors
+                                        Over 150 Contributors
```
Comment (Flyingmana) [inline index.html:514]: cauldron.io reports 228 for the LTS repository alone

File: index.html (hunk 1 above)
Comment (fballiano) [reply index.html:393]: that's what https://github.com/OpenMage/magento-lts/graphs/contributors says, i don't know

File: index.html (hunk 3 above)
Comment (fballiano) [reply index.html:337]: https://github.com/OpenMage/magento-lts/graphs/contributors

File: index.html (hunk 4 above)
Comment (fballiano) [reply index.html:514]: Screenshot 2023-01-06 alle 11 28 51

**Comment by fballiano** [comment]
> is the docker-compose file not working anymore?

I can't use it at the moment

File: index.html (hunk 3 above)
Comment (Flyingmana) [reply index.html:337]: there is also https://github.com/OpenMage/magento-mirror/graphs/contributors and https://github.com/OpenMage/OpenMage.github.io/graphs/contributors

File: index.html (hunk 4 above)
Comment (fballiano) [reply index.html:514]: but also:

Screenshot 2023-01-06 alle 11 30 26

File: index.html (hunk 3 above)
Comment (Flyingmana) [reply index.html:337]: ahh, also I think some of these were in the older branches which now got deleted 🙄

Comment (fballiano) [reply index.html:337]: it becomes very difficult to sum all of the things... personally I'd even remove that data.

Comment (fballiano) [reply index.html]: reverted to 53

File: index.html (hunk 1 above)
Comment (fballiano) [reply index.html]: reverted to 117

File: index.html (hunk 2 above)
Comment (fballiano) [reply index.html:645]: I think I fixed it but I'm not sure, the file I'm editing has an empty line at the bottom :-\

**Review by Flyingmana — APPROVED** [review]

File: index.html (hunk 4 above)
Comment (sreichel) [reply index.html:514]: - 228 seem be all contributors (ignoring name/email) - (i should be listed with at least 3 "accounts")
- 151 added themselve to contributers list
- 177 correct imho

**Pull request merged**